tests/integration/states/test_pkg.py

```diff
@@ -28,12 +29,9 @@
 # Import 3rd-party libs
 from salt.ext import six
 from salt.ext.six.moves import range  # pylint: disable=import-error,redefined-builtin
+from tests.support.sminion import build_minion_opts
 
-try:
-    from distro import LinuxDistribution
-    pre_grains = LinuxDistribution()
-except ImportError:
-    pre_grains = None
+_grains = salt.loader.grains(build_minion_opts(minion_id='runtests-internal-sminion'))
```

Comment:
Please, no dictionaries at the global scope.
If you need them to build the package list, maybe that should actually live in `setUpClass`?

Reply:
On top of being a dictionary, loading grains is not a blazing fast operation and will run just by importing the test module, this alone will slow down pytest test collecting.